docs/README.md

```diff
@@ -22,13 +22,14 @@ You will create/deploy:
 * A stack of OpenFaaS functions via stack.yml
 * Customise limits for Swarm or K8s
 * Setup a container image builder
+* (K8s only) [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) for openfaas and openfaas-fn namespaces
 
 ### Before you begin
 
 * You must enable basic auth to prevent user-functions from accessing the admin API of the gateway
 * A list of valid users is defined in the CUSTOMERS file in this GitHub repo, this acts as an ACL, but you can define your own
 * Swarm offers no isolation between functions (they can call each other)
-* For Kubernetes istolation can be applied through NetworkPolicy
+* For Kubernetes istolation can be applied through [NetworkPolicy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
```

Comment:
Not your fault but `istolation` should be `isolation`

Reply:
Thank you! :-) Didn't see that!